seed/serialize.py

```diff
@@ -213,6 +281,7 @@ def main():
 
     print("Load", args.seed_path.name)
     seed_data = json.load(args.seed_path)
+    seed_data['studies'].sort(key=lambda study: study['name'])
```

Comment:
I suppose python `sort` is stable?
Let's add a comment we can't reorder the study with the same name.

Reply:
why can't we reorder the studies with the same name if they do not overlap by filters?

Reply:
If they don't overlap there is no problem.
If I'm not mistaken overlapped studies with the same name is used in Yandex.

Reply:
> If they don't overlap there is no problem. If I'm not mistaken overlapped studies with the same name is used in Yandex.

We should not allow overlapped studies, added checks ensures we don't.